c/driver/postgresql/statement.cc

```diff
@@ -315,12 +321,13 @@ struct BindStream {
       }
     }
 
-    PGresult* result = PQprepare(conn, /*stmtName=*/"", query.c_str(),
-                                 /*nParams=*/bind_schema->n_children, param_types.data());
-    if (PQresultStatus(result) != PGRES_COMMAND_OK) {
+    const char* temp = "COPY bulk_ingest FROM STDIN WITH (FORMAT binary);";
```

Comment:
I placed this in `Prepare` because it represented a minor diff, but I'm not sure the Prepare/Execute pattern is the same concept as a COPY. Maybe COPY should have a dedicated ADBC method?